indico/modules/search/client/js/components/SearchApp.jsx

```diff
@@ -74,8 +74,27 @@ function SearchTypeMenuItem({index, active, title, total, loading, onClick}) {
   } else if (total !== -1) {
     indicator = ` (${total})`;
   }
+
+  let accessibleLabel = title;
+  if (loading) {
+    accessibleLabel = Translate.string('{title} (loading results)', {title});
+  } else if (!total) {
+    accessibleLabel = Translate.string('{title} (no results)', {title});
+  }
+
   return (
-    <Menu.Item index={index} active={active} onClick={onClick} disabled={loading || !total}>
+    <Menu.Item
```

Comment:
Generally this all works fine, but I'm missing the `cursor: pointer;` now - having the default cursor feels weird, also because pretty much every other button, link, etc. has `cursor: pointer` so I guess this wasn't intentional.

Reply:
Ah ok. I didn't pay attention to that because I figured it would've been taken care of at the library level. 😋 I'll fix it.

Reply:
Added cursor styles for button and label to reset.